app/src/main/kotlin/com/wire/android/ui/common/dialog/StateDialog.kt

```diff
@@ -0,0 +1,15 @@
+package com.wire.android.ui.common.dialog
+
+import androidx.compose.runtime.Composable
+import com.wire.android.model.DialogState
+
+@Composable
+fun <T : Any> StateDialog(
+    dialogState: DialogState<T>,
```

Comment:
`StateDialog`, `DialogState`, it's confusing 😄 Maybe `DialogState` could be renamed to something like `VisibilityState` and be more generic one?

Reply:
Good idea